datumaro/plugins/transforms.py

```diff
@@ -714,6 +715,52 @@ def transform_item(self, item):
         return item.wrap(annotations=annotations)
 
 
+class ProjectInfos(Transform, CliPlugin):
+    """
+    Changes the content of infos.
+    A user can add meta-data of dataset such as author, comments, or related papers.
+    Infos values are not affect on the dataset structure.
+    We thus can add any meta-data freely.
+    """
+
+    @classmethod
+    def build_cmdline_parser(cls, **kwargs):
+        parser = super().build_cmdline_parser(**kwargs)
+        parser.add_argument(
+            "-i",
+            "--infos",
+            action="append",
+            dest="dst_infos",
+            help="A dictionary of the dataset meta-information",
+        )
+        parser.add_argument(
+            "-o",
+            "--over_write",
+            action="store_true",
+            dest="over_write",
+            help="Overwrite the infos of src if True",
+        )
```

Comment:
Can we use `--overwrite` to make it aligned with the existing CLI argument?

https://github.com/openvinotoolkit/datumaro/blob/eb4335c7e5eadf187f5455076afd0dd745b8f1cf/datumaro/cli/commands/convert.py#L68-L70